test/unit/kubernetes-deploy/runner_task_test.rb

```diff
@@ -1,50 +1,27 @@
 # frozen_string_literal: true
 require 'test_helper'
+require 'kubernetes-deploy/runner_task'
 
 class RunnerTaskUnitTest < KubernetesDeploy::TestCase
-  def test_missing_namespace
-    assert_raises(KubernetesDeploy::RunnerTask::FatalTaskRunError,
-      message: "Configuration invalid: Namespace was not found") do
-      task_runner = KubernetesDeploy::RunnerTask.new(
-        context: KubeclientHelper::MINIKUBE_CONTEXT,
-        namespace: "missing",
-      )
-
-      task_runner.run(
-        task_template: 'hello-cloud-template-runner',
-        entrypoint: nil,
-        args: 'a'
-      )
-    end
-  end
-
-  def test_missing_arguments
-    assert_raises(KubernetesDeploy::RunnerTask::FatalTaskRunError) do
-      task_runner = KubernetesDeploy::RunnerTask.new(
-        context: KubeclientHelper::MINIKUBE_CONTEXT,
-        namespace: @namespace,
-      )
-
-      task_runner.run(
-        task_template: 'hello-cloud-template-runner',
-        entrypoint: nil,
-        args: nil
-      )
-    end
+  def setup
+    Kubeclient::Client.any_instance.stubs(:discover)
```

Comment:
If you want unit tests to never hit kubeclient, does it make sense to use `Kubeclient::Client.any_instance.expects(:discover).never`?